### PR TITLE
Medium Complexity Problem6_ReverseString

### PR DESCRIPTION
**Pull request by deepakghadge19**
Added code for Medium Complexity Problem6_ReverseString, so please review it.

### CODE REVIEW

File: src/mediumComplexityProblem6/ReverseString.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package mediumComplexityProblem6;
+
+public class ReverseString {
+    public void reverseStringOperation(char[] inputCharArray) {
```
Comment (amitsahu8) [inline src/mediumComplexityProblem6/ReverseString.java:4]: return the reserve string

File: src/mediumComplexityProblem6/ReverseString.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        int startCounter = 0;
+        int endCounter = inputCharArray.length - 1;
+        while (startCounter < endCounter) {
+            if (!Character.isAlphabetic(inputCharArray[startCounter])) {
```
Comment (amitsahu8) [inline src/mediumComplexityProblem6/ReverseString.java:8]: check for the numeric character also, try to consider edge cases also

File: src/mediumComplexityProblem6/ReverseString.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    public static void main(String[] args) {
+        ReverseString reverseString = new ReverseString();
+        String inputString = "Ro@hitshetty";
+        System.out.println("Input String :" + inputString);
+        char[] chaArray = inputString.toCharArray();
+        reverseString.reverseStringOperation(chaArray);
+        String reverse = new String(chaArray);
+        System.out.println("Reverse output string is :" + reverse);
```
Comment (amitsahu8) [inline src/mediumComplexityProblem6/ReverseString.java:29]: Directly print the return string, no need to store in variable

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: src/mediumComplexityProblem6/ReverseString.java (hunk 1 above)
Comment (deepakghadge19) [reply src/mediumComplexityProblem6/ReverseString.java]: Done!!

File: src/mediumComplexityProblem6/ReverseString.java (hunk 2 above)
Comment (deepakghadge19) [reply src/mediumComplexityProblem6/ReverseString.java]: Done!!

File: src/mediumComplexityProblem6/ReverseString.java (hunk 3 above)
Comment (deepakghadge19) [reply src/mediumComplexityProblem6/ReverseString.java]: Done!!

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**